inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/knowledgebase-profiles.yaml

```diff
@@ -45,8 +44,7 @@ virtuoso:
       description: http://www.schema.org/description
       property-label: http://www.w3.org/2000/01/rdf-schema#label
       property-description: http://www.schema.org/description
-      root-concepts:
-            - http://www.w3.org/2002/07/owl#Nothing
+      root-concepts: [ ]
```

Comment:
If the list is empty, you can probably just leave it out entirely. May require to change the constructor to handle the case that `aRootConcepts` is null an in that case set `rootConcepts` to an empty `ArrayList<>()`.